flink-tests/src/test/java/org/apache/flink/test/checkpointing/EventTimeWindowCheckpointingITCase.java

```diff
@@ -99,11 +110,21 @@
 
 	private AbstractStateBackend stateBackend;
 
+	@Parameterized.Parameter
+	public StateBackendEnum stateBackendEnum;
+
 	enum StateBackendEnum {
 		MEM, FILE, ROCKSDB_FULLY_ASYNC, ROCKSDB_INCREMENTAL, ROCKSDB_INCREMENTAL_ZK, MEM_ASYNC, FILE_ASYNC
 	}
 
-	protected abstract StateBackendEnum getStateBackend();
+	@Parameterized.Parameters(name = "statebackend type ={0}")
+	public static Collection<StateBackendEnum> parameter() {
+		return Arrays.asList(MEM, FILE, ROCKSDB_FULLY_ASYNC, ROCKSDB_INCREMENTAL, ROCKSDB_INCREMENTAL_ZK, MEM_ASYNC, FILE_ASYNC);
```

Comment:
Here we could say `Arrays.asList(StateBackendEnum.values())`.

Reply:
Ok, changed it.